src/transforms/transform-pull-request.ts

```diff
@@ -52,8 +53,8 @@ const mapReviews = async (
 
 			acc.push(usernames[reviewerUsername]);
 
-		} else if (usernames[reviewerUsername].approvalStatus !== STATE_APPROVED && review.state === STATE_APPROVED) {
-			usernames[reviewerUsername].approvalStatus = STATE_APPROVED;
+		} else {
+			usernames[reviewerUsername].approvalStatus = review.state === STATE_APPROVED ? STATE_APPROVED : STATE_UNAPPROVED;
```

Comment:
The else if logic we had before was basically saying if we'd previously seen your review, we don't give a damn if the state has changed... PR APPROVED!